documentation/metadata-editing.md

```diff
@@ -0,0 +1,119 @@
+and other metadata is generated by the server. 
+A `PUT` request replaces the state of the target resource entirely with the request payload.
+This is not possible with description resources and therefore not allowed.
+
+Furthermore, it is also forbidden to perform a `DELETE` on description resources.
+The reason being that a resource will always have some auxiliary description (e.g. content type), 
+which means that it semantically makes no sense to delete description resources.
+
+### Protected metadata
+
+Some metadata is forbidden to be modified.
+The CSS will throw an error (409 `ConflictHttpError`) when trying to change this protected metadata.
+
+## Impact on creating containers
+
+Before editing description resources was possible, it was possible to change the description resource of `ldp:Container`s.
+This was achieved by executing a `PUT` request to the container and adding the metadata triples in the body of that request.
+
+This was not in line with the [Solid Protocol](https://solidproject.org/TR/2021/protocol-20211217#writing-resources)
+about writing resources.
+The protocol states the following:
+
+> Servers MUST NOT allow HTTP POST, PUT and PATCH to update a container’s resource metadata statements;
+> if the server receives such a request, it MUST respond with a 409 status code.
+
+Therefore, since it is now possible to edit description resources directly,
+the above functionality has been removed from the CSS.
+This means that now when creating a container, the contents of the body are ignored.
+Additionally, it is now forbidden to perform a `PUT` request to an existing container.
+
+## Example of a workflow for editing a description resource {#workflow}
+
+In this example, we are trying to add an inbox description to an `ldp:Resource`.
+This inbox indication will then allow discovery for the `ldp:inbox` as described in the [Linked Data Notifications specification](https://www.w3.org/TR/ldn/).
+
+To give some context, assume we have started the CSS with the default configuration
+and have created an inbox which has as URL `http://localhost:3000/inbox/`.
+
+We don't know the location of the description resource
+So first, we send a `HEAD` request to the resource to retrieve the description resource URL.
+
+```shell
+curl --location --head 'http://localhost:3000/'
+```
+
+which will at least the following response
+
+```shell
+HTTP/1.1 200 OK
+Link: <http://localhost:3000/.meta>; rel="describedby"
+```
+Now that we have the URL of the description resource,
+we create a query for adding the inbox in the description of the resource.
+This can be either with a SPARQL Update
+
+```shell
+curl --location --request PATCH 'http://localhost:3000/.meta' \
+--header 'Content-Type: application/sparql-update' \
+--data-raw 'INSERT DATA { <http://localhost:3000/> <http://www.w3.org/ns/ldp#inbox> <http://localhost:3000/inbox/>. }'
```

Comment:
Would simply not have the SPARQL example in this document, only the N3 one (as long as that is the only one in the spec)